tests/utils/test_vep.py

```diff
@@ -0,0 +1,249 @@
+    """Test the filter_vep_consequences_by_loftee function."""
+    # Create a mock ArrayExpression of VEP consequences with LOFTEE annotations
+    csq_expr = hl.literal(
+        [
+            hl.struct(lof="HC", lof_flags=hl.missing(hl.tstr)),
+            hl.struct(lof="HC", lof_flags=""),
+            hl.struct(lof="LC", lof_flags="flag1"),
+            hl.struct(lof="OS", lof_flags=hl.missing(hl.tstr)),
+            hl.struct(lof=hl.missing(hl.tstr), lof_flags="flag2"),
+        ]
+    )
+
+    # Test with default parameters
+    result = filter_vep_transcript_csqs_expr(csq_expr)
+    assert hl.eval(hl.len(result)) == 5, "Expected 5 consequences"
+
+    # Test with loftee_labels
+    result = filter_vep_transcript_csqs_expr(csq_expr, loftee_labels=["HC"])
+    assert hl.eval(hl.len(result)) == 2, "Expected 2 consequences"
+    assert hl.eval(result[0].lof) == "HC", "Expected 'HC'"
+
+    # Test with no_lof_flags
+    result = filter_vep_transcript_csqs_expr(csq_expr, no_lof_flags=True)
+    assert hl.eval(hl.len(result)) == 3, "Expected 3 consequences"
+    assert hl.eval(
+        hl.all(
+            result.map(lambda csq: hl.is_missing(csq.lof_flags) | (csq.lof_flags == ""))
+        )
+    ), "Expected no LOFTEE flags"
+
+    # Test with loftee_labels and no_lof_flags
+    result = filter_vep_transcript_csqs_expr(
+        csq_expr, loftee_labels=["HC"], no_lof_flags=True
+    )
+    assert hl.eval(hl.len(result)) == 2, "Expected 2 consequence"
+    assert hl.eval(result[0].lof) == "HC", "Expected 'HC'"
+    assert hl.eval(
+        hl.all(
+            result.map(lambda csq: hl.is_missing(csq.lof_flags) | (csq.lof_flags == ""))
+        )
+    ), "Expected no LOFTEE flags"
+
+
+def test_get_most_severe_csq_from_multiple_csq_lists():
+    """Test the get_most_severe_csq_from_multiple_csq_lists function."""
+    # Create a mock VEP expression
+    vep_expr = hl.struct(
+        transcript_consequences=[
+            hl.struct(
+                biotype="protein_coding",
+                lof="HC",
+                lof_flags="flag1",
+                consequence_terms="missense_variant",
+            ),
+            hl.struct(
+                biotype="protein_coding",
+                lof="HC",
+                lof_flags=hl.missing(hl.tstr),
+                consequence_terms="synonymous_variant",
+            ),
+            hl.struct(
+                biotype="protein_coding",
+                lof="HC",
+                lof_flags=hl.missing(hl.tstr),
+                consequence_terms="missense_variant",
+            ),
+        ],
+        intergenic_consequences=[
+            hl.struct(
+                biotype="intergenic",
+                lof=hl.missing(hl.tstr),
+                lof_flags=hl.missing(hl.tstr),
+                consequence_terms="intergenic_variant",
+            )
+        ],
+    )
+
+    # Test the function
+    result = get_most_severe_csq_from_multiple_csq_lists(vep_expr)
```

Comment:
also tested with setting prioritize_loftee_no_flags to True or False -- does it make sense that got the same result regardless of how set this param was set?